ukrainian/take_almost_all/take_almost_all.md

```diff
@@ -0,0 +1 @@
+[Беру з собою мотузку, запальничку, ніж та виделку і стрибаю через стіну!](https://github.com/udacity/create-your-own-adventure/blob/master/ukrainian/break-through-wall/break-through-wall.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.